jetty-server/src/test/java/org/eclipse/jetty/server/RequestTest.java

```diff
@@ -1790,6 +1790,17 @@ public void testAmbiguousEncoding() throws Exception
         _connector.getBean(HttpConnectionFactory.class).getHttpConfiguration().setUriCompliance(UriCompliance.UNSAFE);
         assertThat(_connector.getResponse(request), startsWith("HTTP/1.1 200"));
     }
+
+    @Test
+    public void testDoubleSlash() throws Exception
+    {
+        _handler._checker = (request, response) -> true;
+        String request = "GET //../foo HTTP/1.0\r\n" +
```

Comment:
This is a strange test. It has an authority of `..` and a path of `/foo`.  
Why is it a bad request?   Can you add a comment saying why it is bad and check that is really why a 400 is received

Reply:
Ahh yes I need to fix this, I was just using it for debugging and forgot about it.
I will need to test out the compliance modes here.